steps/pcloudy-app-upload/0.1.2/step.yml

```diff
@@ -0,0 +1,63 @@
+title: pCloudy App Upload
+summary: |
+  Uploads your APK or IPA to Pcloudy for interactive app testing.
+description: |
+  Uploads your APK or IPA to Pcloudy for interactive app testing. Please signup to get more details https://www.pcloudy.com
+website: https://github.com/shibupanda/bitrise-step-pcloudy-upload-app
+source_code_url: https://github.com/shibupanda/bitrise-step-pcloudy-upload-app
+support_url: https://github.com/shibupanda/bitrise-step-pcloudy-upload-app/issues
+published_at: 2021-06-25T12:43:43.472427114+05:30
+source:
+  git: https://github.com/shibupanda/bitrise-step-pcloudy-upload-app.git
+  commit: 4f5d221ec046e1c9a3a180dc14079cdde404ab74
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
```

Comment:
it seems the Step Input key (`upload_path `) was removed here, without the default value it should look like:

```
- upload_path:
  opts:
     description: |
       The app file you want to upload to Pcloudy, usually $BITRISE\_APK\_PATH or $BITRISE\_IPA\_PATH.
     is_expand: true
     is_required: true
     summary: Location of the IPA or APK that you want to upload.
     title: Location of APK or IPA
```